src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -42,20 +42,43 @@ export const writePackageTransform: Transform = transformFromPromise(async graph
     );
   }
   if (ngPackage.assets.length && !ngEntryPoint.isSecondaryEntryPoint) {
-    const assets = ngPackage.assets.map(x => path.join(ngPackage.src, x));
+    const ingoreOptions = {
+      ignore: ['.gitkeep', '**/.DS_Store', '**/Thumbs.db', '**/node_modules/**', `${ngPackage.dest}/**`],
```

Comment:
Why ignore the node_modules? This would be really useful for me because I use another library that comes with its own assets that need to be delivered with my library.